### ISSUE TITLE
Could not find SemEval tree file

### ISSUE THREAD

**Issue by l6chen** [morningmoni/TaxoRL#8]
As you wrote in train_RL.py:
```
        trees_semeval = read_edge_files("../datasets/SemEval-2016/original/",
                                        given_root=True, filter_root=args.filter_root, allow_up=False)
```
It seems you are trying to read tree.ptb from a directory. So should there be a tree file for semEval and how could I find/generate it?

**Comment by morningmoni** [comment]
Are the files not there? The format is different from WordNet files (not *.ptb). https://github.com/morningmoni/TaxoRL/blob/master/datasets/SemEval-2016/original/science_en.taxo

**Comment by l6chen** [comment]
OK. I see. And did you use the parse_wikipedia.py to parse other corpus as you write in readme and in your paper? I used the resource file (containing .db files) from the LexNet, but it turns out overfitting. 
Plus where did you find the wordnet tree.ptb?

**Comment by morningmoni** [comment]
What does "overfitting" mean here?

http://www.cs.unc.edu/~mbansal/codedata/bansal-taxo-generalsetup.zip

**Comment by l6chen** [comment]
Overfitting there means the metric in the training step keep going up while the metric in the testing step going down.

**Comment by morningmoni** [comment]
Yes we did. The original resource file may miss terms of interest.
